CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## v0.138.5, 26 March 2021
+
+- Update maven/spec/dependabot/maven/version_spec.rb
+- Update gradle/spec/dependabot/gradle/version_spec.rb
```

Comment:
```suggestion
```

Idea: maybe these are covered by the `Maven/Gradle: Treat dev and pr as pre-releases for gradle/maven` below?